x/sigs/codec.proto

```diff
@@ -25,3 +25,12 @@ message StdSignature {
   // Removed Address, Pubkey is more powerful
   crypto.Signature signature = 4;
 }
+
+// BumpSequenceMsg increments a sequence counter by given amount for a user.
+message BumpSequenceMsg {
+  // Public key of the user.
+  crypto.PublicKey pubkey = 1;
```

Comment:
q: What use cases do you have in mind where the pubKey is not the main signer? This adds quite some complexity

Reply:
I can remove it then. I thought being explicit here makes it more obvious.